### PR TITLE
[MM-21381] Add unit tests to the "SearchTeamCmd" mmctl command

### PR DESCRIPTION
**Pull request by phillipahereza**
#### Summary
Covers the addition of unit tests to the `SearchTeamCmd` command, in the `commands/team.go` file

#### Ticket Link

Fixes: https://mattermost.atlassian.net/browse/MM-21381

### CODE REVIEW

File: commands/team.go [hunk 1]
```diff
@@ -314,8 +315,12 @@ func searchTeamCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 
 	sortedTeams := removeDuplicatesAndSortTeams(teams)
 
-	for _, team := range sortedTeams {
-		printer.PrintT("{{.Name}}: {{.DisplayName}} ({{.Id}})", team)
+	if len(sortedTeams) > 0 {
```
Comment (agnivade) [inline commands/team.go]: I would suggest to make this similar to `searchUserCmdF`. Move the check to the loop above. And before appending, check if len(foundteams) is 0, then print - `"Unable to find team '"+ searchTerm "'"`

File: commands/team_test.go [hunk 2]
```diff
@@ -614,3 +612,184 @@ func (s *MmctlUnitTestSuite) TestRemoveUserCmd() {
...
+			Times(1)
+
+		err := searchTeamCmdF(s.client, &cobra.Command{}, []string{mockTeam1Name, mockTeam2DisplayName})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetErrorLines(), 0)
+		s.Require().Len(printer.GetLines(), 2)
+		s.Require().Equal(mockTeam1, printer.GetLines()[0])
+		s.Require().Equal(mockTeam2, printer.GetLines()[1])
+	})
+	//
```
Comment (agnivade) [inline commands/team_test.go]: Stray comment. Let's remove this.

File: commands/team_test.go [hunk 3]
```diff
@@ -614,3 +612,184 @@ func (s *MmctlUnitTestSuite) TestRemoveUserCmd() {
...
+		err := searchTeamCmdF(s.client, &cobra.Command{}, []string{teamVariableName})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetErrorLines(), 0)
+		s.Require().Len(printer.GetLines(), 2)
+		s.Require().Equal(mockTeam1, printer.GetLines()[0])
+		s.Require().Equal(mockTeam2, printer.GetLines()[1])
+
+	})
+
+	s.Run("Test duplicates removed from search results", func() {
```
Comment (agnivade) [inline commands/team_test.go]: Test duplicates _are_ removed from search results

File: commands/team_test.go [hunk 4]
```diff
@@ -2,10 +2,8 @@ package commands
 
 import (
 	"errors"
-
```
Comment (agnivade) [inline commands/team_test.go]: I think these were unintentional. Let's undo them.

**Comment by hanzei** [comment]
Hey @phillipahereza,

Looks like this PR is almost ready to merge. Do you have any questions about the feedback at https://github.com/mattermost/mmctl/pull/90#pullrequestreview-337631334?

**Comment by phillipahereza** [comment]
@hanzei sorry I had forgotten about this, I'll implement the requested changes

**Comment by jespino** [comment]
Merged!, Thanks @phillipahereza!

**Pull request merged**